config.go

```diff
@@ -50,14 +50,21 @@ func DefaultConfig(authToken string) ClientConfig {
 	}
 }
 
-func DefaultAzureConfig(apiKey, baseURL, engine string) ClientConfig {
+func DefaultAzureConfig(apiKey, baseURL string, modelMapper map[string]string) ClientConfig {
```

Comment:
I suggest we remove `modelMapper` argument here as it doesn't really fit the "default" config. Let's allow modifying it afterward.

Reply:
OK